src/wwwroot/app/main.js

```diff
@@ -81,7 +83,8 @@ angular
    .directive('contacts',  ContactInfoDirective.createInstance)
    .directive('ngThumb',   CanvasPreviewDirective)
 
-   .filter('tel', phoneFormatFilter)
+   .filter('tel',     PhoneFormatFilter)
+   .filter('arrShow', ArrayShowFilter)
```

Comment:
change name. I think something like  `arrayAsString` is more correct. U can change to another